src/wazuh_db/schema_upgrade_v8.sql

```diff
@@ -0,0 +1,33 @@
+/*
+ * SQL Schema for upgrading databases
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ *
+ * March 16, 2021.
+ *
+ * This program is a free software, you can redistribute it
+ * and/or modify it under the terms of GPLv2.
+*/
+
+ALTER TABLE sys_osinfo ADD COLUMN reference TEXT DEFAULT '' NOT NULL;
+ALTER TABLE sys_osinfo ADD COLUMN triaged INTEGER(1) DEFAULT 0;
+
+DROP TABLE IF EXISTS vuln_cves;
```

Comment:
We are assuming here that the old vuln_cves table will be completely eliminated. Right? We have to analyze how can this impact on the baseline mechanism.

Reply:
+1

Reply:
The baseline scan was considered by cleaning the last scan column in `vuln_metadata`:
 ` ` `UPDATE vuln_metadata SET LAST_SCAN = '0'; ` ` `